lua/rzls/handlers/providehtmldocumentcolor.lua

```diff
@@ -0,0 +1,39 @@
+local documentstore = require("rzls.documentstore")
+local razor = require("rzls.razor")
+local Log = require("rzls.log")
+
+local empty_response = {}
+
+---@param _err lsp.ResponseError
+---@param result lsp.DocumentDiagnosticParams
+---@param _ctx lsp.HandlerContext
+---@param _config table
+return function(_err, result, _ctx, _config)
+    assert(not _err, vim.inspect(_err))
+    local virtual_document =
+        documentstore.get_virtual_document(result.textDocument.uri, razor.language_kinds.html, "any")
+    assert(virtual_document, "html document was not found")
+    local virtual_client = virtual_document:get_lsp_client()
+    assert(virtual_client, "Could not find LSP client for virtual document")
```

Comment:
We should remove some of those `assert`s in favor of returning an empty response. This can cause some issues when the LSP is not attached yet.

@tris203 is already removing some of the `assert`s we used in a another PR: https://github.com/tris203/rzls.nvim/pull/16/commits/89d3be0b04948d8fc22a8bf55d6b0cf979eda8ee